frontend/test/__support__/commands.js

```diff
@@ -134,6 +133,42 @@ Cypress.Commands.add("updateCollectionGraph", (groupsCollectionObject = {}) => {
   );
 });
 
+Cypress.Commands.add(
+  "sandboxTable",
+  ({
+    attribute_remappings = {},
+    card_id = null,
+    group_id = 4,
+    table_id = 2,
+  } = {}) => {
+    // Extract the name of the table, as well as `schema` and `db_id` that we'll need later on for `cy.updatePermissionsSchemas()`
+    cy.request("GET", "/api/table").then(({ body: tables }) => {
+      const [{ name, schema, db_id }] = tables.filter(table => {
+        return table.id === table_id;
+      });
```

Comment:
```suggestion
      const { name, schema, db_id } = tables.find(table => table.id === table_id);
```

Reply:
Oh, this is nice - I like it! Thank you :)